WalletWasabi.Fluent/ViewModels/Wallets/Send/PrivacySuggestionsFlyoutViewModel.cs

```diff
@@ -34,8 +34,10 @@ public PrivacySuggestionsFlyoutViewModel()
 
 	public ObservableCollection<SuggestionViewModel> Suggestions { get; }
 
-	public async Task BuildPrivacySuggestionsAsync(Wallet wallet, TransactionInfo info, BitcoinAddress destination, BuildTransactionResult transaction, bool isFixedAmount, CancellationToken cancellationToken)
+	public async Task BuildPrivacySuggestionsAsync(Wallet wallet, TransactionInfo info, BitcoinAddress destination, BuildTransactionResult transaction, bool isFixedAmount)
 	{
+		using CancellationTokenSource cancellationTokenSource = new(TimeSpan.FromSeconds(15));
```

Comment:
When you add `using` like this, you should not have L73 and L74 present (double dispose).

Reply:
Thx nice catch! I was experimenting with it, and left both of them in accidentally.